index.js

```diff
@@ -52,9 +51,9 @@ function transformEventToRow(fullEvent) {
     console.log(properties)
     // only move prop to elements for the $autocapture action
     if (event === '$autocapture' && properties?.['$elements']) {
```

Comment:
@yakkomajuri  we are not exactly sure what needs to be done here could you give an example.

Reply:
this comment:

```
    // only move prop to elements for the $autocapture action
```
was written by you - not me :D